.circleci/config.yml

```diff
@@ -5,10 +5,10 @@ orbs:
     executors:
       node:
         docker:
-          - image: cimg/node:16.20.2
+          - image: cimg/node:18.18.2
```

Comment:
feels like we can just use `18.20` LTS release?

Reply:
The reasoning here was to keep the same as what `nvm use` yielded for me, though I probably had a cached 18.x version:

<img width="644" alt="image" src="https://github.com/user-attachments/assets/ff66ff4c-8150-465f-8cbf-3e8afbe5896b">

However I `nvm uninstall` and `nvm install`'d again and looks like 18.20.4 is the one that's going to be installed currently:

<img width="698" alt="image" src="https://github.com/user-attachments/assets/be4507f1-2d1c-407e-a8e9-5699293fcfce">

Might as well go with [current LTS](https://nodejs.org/download/release/latest-v18.x/)'s latest minor then! bb39b7972ba456de8c4613582ce8339b8b3d30bb